set.mm

```diff
@@ -50971,6 +50977,16 @@ the restriction (of the relation) to the singleton containing this
       CTUDUEUFVCUTVADLDAUGUTVADUHUKTUIUJ $.
   $}
 
+  ${
+    $d x A $.
+    $( Characterization of the elements of the identity relation.  (Contributed
+       by BJ, 28-Aug-2022.) $)
+    elid $p |- ( A e. _I <-> E. x A = <. x , x >. ) $=
+      ( cid wcel cvv cres cv cop wceq wrex wex wrel reli residOLD eqcomi eleq2i
```

Comment:
Why cannot ~dfrel3 be used here? If ~residOLD is really needed, then ~residOLD cannot be obsolete

Reply:
~dfrel3 is after ~elid so cannot be used at this stage. I think it is possible to move theorems around to have dfrel3 before elid and therefore use it, but that subsection is kind of a mess, because many definitions are introduced "en masse". Actually, I decided to first try to reorder a bit that subsection (in a following PR) and then resume work related to the identity relation. I'll put a TODO in ~elid to tell to move reorder statements and use dfrel3.

Reply:
That's fine, thanks.